### ISSUE TITLE
D2BS is not responding... starting profile

### ISSUE THREAD

**Issue by cobbleloo** [kolton/d2bot-with-kolbot#534]
Hi,

I'm trying to run a bot on a virtual machine. However, when I click Start, the game opens and sits there for ~15 seconds before giving me "D2BS is not responding... starting profile" in the Console on the D2Bot window. Then it tries to restart and loops like that until I stop or exit.

The machine is Hyper-V Windows 10 Pro 64-bit guest running on a Windows 10 Pro 64-bit host.

Config and Starting script are OK as far as I can tell. I have tested them on the host computer with no problems. Everything is basically the same on the guest as it is on the host, as far as settings, but it runs fine on the host and won't work on the guest.

Any help would be appreciated, thanks.

**Comment by krazy03** [comment]
Did put d2bs manager to run in windows 7 compatibility mode and as administrator??

Also this is just my thought if you went through the trouble of getting  VM why not go with windows XP? It's so much better. I used to run some on windows 10 and randomly i would get "D2BS is not responding... starting profile" like every other run. Which is why i went for a XP VM, 100% problem free.

**Comment by cobbleloo** [comment]
Yeah I tried all the compatibility combinations. This is just the vm I had already set up.

**Comment by MightyToSaveGod** [comment]
_**I've noticed my home key isn't working for kolbot the button "home." I'm getting the error msg "D2BS is not responding... starting profile" #i believe it has something to do with my home key not loading. for kolbot usually when i go to load a profile and run the bot "the home key works." usually if the home key works i don't see that error msg "D2BS is not responding... starting profile" this leads me to believe the home key not loading/working properly is the reason for why I'm seeing that error msg #any advice suggestions thoughts what I'm not doing correctly for "home key to not work on both my keyboard and laptop." it will not load/run/work on kolbot the button "home." does not load. and i believe that's the reason why I'm seeing that error msg "D2BS is not responding... starting profile"**_
